docs/metrics.md

```diff
@@ -157,29 +157,41 @@ Fired upon successful submission of the survey containing the survey data augmen
 - `timerFirstPaint`: the number of milliseconds between the start of the page load and the first paint.
 - `timerFirstInteraction`: the number of milliseconds between the start of the page load and the first of the following events: `click`, `touch`, `scroll`, `keypress`. `null` if none of these events took place.
 - `timerFirstByte`: the number of milliseconds between the start of the request and the time the first byte was received.
-- `requests`: an object containing information about requests spawned by the page. Each member key indicates the type of request, categorized in the same way as the [Firefox Developer Tools Network Monitor](https://developer.mozilla.org/docs/Tools/Network_Monitor):
-
-  * `html`: HTML files
-  * `css`: Cascading style sheets
-  * `js`: JavaScript
-  * `xhr`: files retrieved by `XMLHttpRequest`
-  * `fonts`: non-system fonts
-  * `images`: image files
-  * `media`: audio and video files
-  * `flash`: Flash embeds
-  * `ws`: WebSocket connections
-  * `other`: anything not covered in the above categories
-  * `all`: all requests summed.
-
+- `requests`: an object containing information about requests spawned by the page. Each member key indicates the type of request, categorized as [`webRequest.ResourceType`s](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/webRequest/ResourceType):
+
+  * `main_frame`
+  * `sub_frame`
+  * `stylesheet`
+  * `script`
+  * `image`
+  * `object`
+  * `xmlhttprequest`
+  * `xbl`
+  * `xslt`
+  * `ping`
+  * `beacon`
+  * `xml_dtd`
+  * `font`
+  * `media`
+  * `websocket`
+  * `csp_report`
+  * `imageset`
+  * `web_manifest`
```

Comment:
Not sure if these are in a special order, or if you want to A-z them (minus "other" and "all") for easier readability/scanability. (Plus, I just really enjoy being annoying)